zap/src/test/java/org/parosproxy/paros/core/scanner/AbstractPluginUnitTest.java

```diff
@@ -63,6 +66,14 @@
 /** Unit test for {@link AbstractPlugin}. */
 class AbstractPluginUnitTest extends PluginTestUtils {
 
+    private static final List<String> METHODS_NO_ENCLOSED_CONTENT =
+            Arrays.asList(
```

Comment:
Can we start using Java 9+ features now? If so, this could be replaced with `List.of(...)`.

Reply:
Not yet.